java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -6691,4 +6692,56 @@ void testApplyBooleanMaskFromListOfStructure() {
       assertColumnsAreEqual(expectedCv, actualCv);
     }
   }
+
+  private ColumnView getColumnViewWithNonEmptyNulls() {
+    List<Integer> list0 = Arrays.asList(1, 2, 3);
+    List<Integer> list1 = Arrays.asList(4, 5, null);
+    List<Integer> list2 = Arrays.asList(7, 8, 9);
+    List<Integer> list3 = null;
+    ColumnVector input =  makeListsColumn(DType.INT32, list0, list1, list2, list3);
```

Comment:
CI failed because there is memory leak in Java tests. Sorry I overlooked the Java tests. Please fix them.

Reply:
`ColumnVector` as well as `HostColumnVector` must be wrapped in `try` blocks.

Reply:
Hmm... makes me want to have this failure on my local so we can avoid this. It will still be good to get to the bottom of this discrepancy. 

In the meanwhile, I will push a fix. Thanks for pointing us to this.

Reply:
Agreed, the discrepancy isn't ideal.

Could it be caused by the types of GPUs that are used in CI vs. on our local machines?

I'm wondering if that or the amount of memory on a given GPU would contribute to this.

Reply:
Maybe there is a variable that controls the test behavior, and that var is set by the local environment. @revans2 Do you know about this?

Reply:
If a memory leak happens and java's GC is fast, then the leak is cleared up before RMM is shut down and there is no error.  If GC is slow, then the leak is not cleared and you get the error.  We have a flag to print out messages to help debug leaks, but it is off by default because it makes the tests take a lot longer.  Just set `-Dai.rapids.refcount.debug=true` on the maven commend line when running the tests. Even if they pass you can then look at the logs to see if any leaks were detected and use the stack traces to track down what is leaked and why.

Reply:
But in this case aren't the vectors off the heap? Did you mean the MemoryCleaner instead of GC?